adding-a-new-module/template/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+val ext = (gradle as ExtensionAware).extra
+
+apply(from = "../gradle/publishable_component.gradle")
+
+group = "com.splendo.kaluga"
+version = ext["library_version"]!!
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+dependencies {
+    /* Uncomment this line if you are using mockito
+     testImplementation("org.mockito:mockito-core:2.28.2")
+    */
+
+    /* Uncomment these lines if you are using fragments
+    implementation("androidx.fragment:fragment:1.1.0")
```

Comment:
hardcoded version,

Since we will copy this everywhere can be simplified to `implementation androidx.fragment:fragment-ktx:$whatever`, that should have `androidx.fragment:fragment` as a dependency and are strongly promoting kotlin.

Reply:
As I understand "androidx.fragment:fragment-ktx" used for test implementation only so I prefer to leave it there

Reply:
No it stands for *K*o*t*lin E*x*tensions. There are many of these, they add some nice kotlin methods to java based packages. 

Someone needed it in a test so added it in the test scope, then everyone copy pasted it.

Reply:
Okey. I'll remove it

Reply:
Ok, for the record I did not suggest removing it.

Reply:
Okey. It's commented anyway ...

Reply:
My suggestion (first comment) is to simplify it to a single line:

`androidx.fragment:fragment-ktx:$whatever`

Reply:
Done